src/server/actions/Organizations.ts

```diff
@@ -1,6 +1,25 @@
 import dbConnect from '@/utils/db-connect';
 import OrganizationSchema from '../models/Organization';
 import { Organization } from '@/types/dataModel/organization';
+import { CreateOrganizationRequest } from '@/types/dataModel/organization';
+
+/**
+ * Create an organization
+ * @param CreateOrganizationRequest requires the name of the organization
+ * @returns _id The Id of the organization
+ */
+
+export async function createOrganization(
+  request: CreateOrganizationRequest
+): Promise<string> {
+  try {
```

Comment:
i am not sure we want to catch the error here, I believe we will want to catch any errors in the api that way we can know what caused it in the API and return the appropriate response code. what do you think @andlrutt

Reply:
I have tested to see what happens when try/catch(error) is not there. The create() function fails and throws an error message. Are we sure we do not want any error handling?

Reply:
wait so you are unable to create organizations at all?

Reply:
You are able to create organizations if the right parameters are given in either conditions.

Reply:
Just discussed with @zaviermiller, here's a proposed pattern:
1. Yes catch in the server function, but just throw again. Be as granular with the errors that we catch (duplicate entity, database connection fail, etc) and then we can control what error messages are thrown (cybersecurity rule, never pass thrown errors directly to the caller :))
2. In the API, try/catch again, and then we can control the status codes based on what the server function threw. For now it will be pretty primitive, but we'll probably get a story out soon to set up a generalized error handling system